README.md

```diff
@@ -271,6 +271,20 @@ ENV['ember-a11y-testing']: {
 }
 ```
 
+The color contrast check can cause undesired scrolling behavior.  To disable:
```

Comment:
We can combine the examples (which was what I meant by my above suggestion, I guess I was just unclear).

The purpose of this document should be to show how to use the features provided by ember-a11y-testing, which, in this case, amounts to passing options through to the underlying axe-core library. We should provide an example of how exactly to specify the options and then defer to the [axe-core documentation for specifics](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter).